docs/aws_freertos_port.md

```diff
@@ -0,0 +1,58 @@
+# AWS FreeRTOS Port
+
+## The Porting Process
+
+Porting from AWS FreeRTOS applications can be done using the following sections. There are several topics to cover including building, porting, authentication, and feature set.
```

Comment:
lets replace all the places of AWS with Amzon FreeRTOS

Reply:
yup good call i like it